pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -467,24 +539,35 @@ func TestGenericScheduler(t *testing.T) {
 			name: "deleted PVC",
 			wErr: fmt.Errorf("persistentvolumeclaim \"existingPVC\" is being deleted"),
 		},
-		{
-			// alwaysCheckAllPredicates is true
-			predicates:               map[string]algorithmpredicates.FitPredicate{"true": truePredicate, "matches": matchesPredicate, "false": falsePredicate},
-			alwaysCheckAllPredicates: true,
-			nodes:                    []string{"1"},
-			pod:                      &v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "2", UID: types.UID("2")}},
-			name:                     "test alwaysCheckAllPredicates is true",
-			wErr: &FitError{
-				Pod:         &v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "2", UID: types.UID("2")}},
-				NumAllNodes: 1,
-				FailedPredicates: FailedPredicateMap{
-					"1": []algorithmpredicates.PredicateFailureReason{algorithmpredicates.ErrFakePredicate, algorithmpredicates.ErrFakePredicate},
-				},
-				FilteredNodesStatuses: framework.NodeToStatusMap{},
+		// TODO(Huang-Wei): remove this when 'alwaysCheckAllPredicates' gets removed from API.
```

Comment:
remove it now? alwaysCheckAllPredicates does not have an impact now anyways.

Reply:
SG.